pkg/kubelet/metrics/metrics.go

```diff
@@ -938,6 +942,18 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DRASubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Duration in seconds (per pod) of the DRA operations (PrepareResources and UnprepareResources).",
```

Comment:
I don't see a reason for putting "per pod" into brackets. How about this?

> Duration in seconds used for preparing or unpreparing all ResourceClaims of a pod. Tracked separately for successful and unsuccessful operations. The total number of failed operations is provided through the histogram's overall count.

Reply:
Remember, this is the user-visible documentation that shows up on https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/

Looking at that, I actually like the wording from `apiserver_admission_controller_admission_duration_seconds` better. Applied to this one here:

> Latency histogram in seconds for the duration of handling all ResourceClaims referenced by a pod when the pod starts or stops. Identified by the name of the operation (PrepareResources or UnprepareResources) and separated by the success of the operation. The number of failed operations is provided through the histogram's overall count.

Reply:
done